util/cron/test-gasnet-asan.bash

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Test ASAN-compatible configuration with gasnet & ASAN on linux64, running
+# multilocale tests only.
+#
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-gasnet.bash
+source $CWD/common-asan.bash
+source $CWD/common-localnode-paratest.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="gasnet-asan"
+
+export GASNET_QUIET=Y
+
+# "Should be able to detect most memory errors, since remote r/w are
+# performed using active messages instead of RDMA".
```

Comment:
Is this about the udp segment? Can you clarify?

Reply:
https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/Sanitizers.rst#limitations

Page mentions that UDP/everything is best, so I thought it might be good to specifically enforce that (though we might just default to this anyway, wasn't certain).

Reply:
Then, I think we should put the path to that file in the comment.

Reply:
I clarified the comment, let me know if it is better.